pivotal-cf/pks-kubo-release
# Feature/osl collector

**What this PR does / why we need it**:
Adds a package which compiles some Go code and outputs osl-output.json file 

**How can this PR be verified?**
Deploy this to a BOSH director with a job which uses the osl-collector package, and verify that `/var/vcap/packages/osl-collector/osl-output.json` exists

**Release note**:
```release-note
Adds open source licensing information upon compilation
```
